readme.md

```diff
@@ -217,6 +217,14 @@ Yields, in [hast][] (**note**: the `pre` and `language-js` class are normal
 }
 ```
 
+### `all(h, parent)`
+
+Helper function for writing custom handlers passed to `options.handlers`.
```

Comment:
```suggestion
Helper function for writing custom handlers passed to `options.handlers`.
Pass it `h` and a parent node (mdast) and it will turn the node’s children into
an array of transformed nodes (hast).
```